### PR TITLE
review fix from #8600

### PR DESCRIPTION
**Pull request by DanielOsypenko**
fix setup - move set_osd_op_complaint_time to fixtures

### CODE REVIEW

**Comment by DanielOsypenko** [comment]
test PASS
https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/30317/console

**Comment by DanielOsypenko** [comment]
@ebenahar the fix was requested on cherry-picks #8600 in comments. Please validate and I will backport it.

File: tests/conftest.py [hunk 1]
```diff
@@ -6914,3 +6914,35 @@ def setup_logwriter_rbd_workload_factory(request, project_factory, teardown_fact
...
+        """
+        Set default values for:
+          osd_op_complaint_time=30.000000
+        """
+        # set the osd_op_complaint_time to selected monitor back to default value
+        ct_pod.exec_ceph_cmd(
+            f"ceph config set osd osd_op_complaint_time {constants.DEFAULT_OSD_OP_COMPLAINT_TIME}"
+        )
+
+    request.addfinalizer(finalizer)
```
Comment (petr-balogh) [inline tests/conftest.py:6946]: I would move it before the line 6932

**Comment by DanielOsypenko** [comment]
to update

**Pull request closed without merging**